app/views/assets/view.php

```diff
@@ -0,0 +1,147 @@
+            <ul class="nav navbar-nav">
+            </ul>
+
+            <ul class="nav navbar-nav navbar-right">
+                <!--li><a href="#">Link</a></li-->
+                <li>
+                    <a href="#">
+                        <i class="glyphicon glyphicon-edit"></i>
+                        Sign Up
+                    </a>
+                </li>
+                <li>
+                    <a href="#">
+                        <i class="glyphicon glyphicon-log-in"></i>
+                        Sign In
+                    </a>
+                </li>
+                <li>
+                    <a href="#">
+                        <i class="glyphicon glyphicon-info-sign"></i>
+                        Help
+                    </a>
+                </li>
+            </ul>
+
+            <ul class="nav navbar-nav navbar-left">
+                <li>
+                    <a href="#menu-toggle" id="menu-toggle">
+                        Modules
+                    </a>
+                </li>
+            </ul>
+
+        </div><!-- /.navbar-collapse -->
+    </div><!-- /.container-fluid -->
+</nav>
+<body style="padding: 50px">
+<div id="wrapper" class="toggled">
+    <!-- Sidebar -->
+    <div id="sidebar-wrapper">
+        <ul class="sidebar-nav">
+
+            <li>
+                <a href="<?php echo base_url(); ?>missing/">
+                    Missing People
+                </a>
+            </li>
+
+            <li>
+                <a href="<?php echo base_url(); ?>organisations/">
+                    Organisations
+                </a>
+            </li>
+
+            <li>
+                <a href="#">
+                    Camps
+                </a>
+            </li>
+
+            <li>
+                <a href="<?php echo base_url(); ?>assets/">
+                    Assets
+                </a>
+            </li>
+
+            <li>
+                <a href="#">
+                    Requests
+                </a>
+            </li>
+
+        </ul>
+    </div>
+    <div id="page-content-wrapper" style="background-color: #ffffff;">
+        <div class="container-fluid">
+            <div class="row">
+                <div class="col-lg-12">
+                    <div class="page-header">
+                        <h1 style="text-align: center;">
+                            Assets Available<br/>
+                        </h1>
+                        <br /><br />
+                    </div>
+                    <?php
+                        foreach ($assets_list as $assets) {
+                            echo "<div class='panel panel-primary'>
+                                      <div class='panel-heading' style='font-size: large; padding-bottom: 15px'>
+                                        <b>Asset Name: " . $assets['name'] . "</b>
+                                        <div class='btn btn-info show' style='float: right;'>Show More</div>
+                                      </div>";
```

Comment:
Just a suggestion. It is lot of html written in PHP. We could also close php tag and write this without `echo`-ing.